### PR TITLE
MSEX v1.2 additions

### PR DESCRIPTION
**Pull request by MrRoundRobin**
- Fixed PName naming
- Added MSEX/CInf supported versions
- Added MSEX/SInf 1.2
- Added UUID field in 1.2 pakets
- Added MSEX/ELIn 1.2
- Added MSEX/MEIn 1.2
- Added MSEX/GEIn 1.2
- Added MSEX/GETh 1.2

### CODE REVIEW

**Comment by hossimo** [comment]
Thanks for the PR, looks good though it's been some time since I looked at this code and I need some testing with MSEX 1.2. May I ask what you are testing with?

**Comment by MrRoundRobin** [comment]
currently i made the changes only according to the specification

i want to add full 1.2 support because of an other project where i want to add CITP to an media server

File: citp.lua [hunk 1]
```diff
@@ -164,9 +180,83 @@ function citp_proto.dissector(buffer,pinfo,tree)
...
+        if bit.band(supported_types,00000064) > 0 then
+          str = str .. lt[7] .. ", "
+        end
+        if bit.band(supported_types,00000128) > 0 then
+          str = str .. lt[8] .. ", "
+        end
+        if bit.band(supported_types,00000256) > 0 then
+          str = str .. lt[9] .. ", "
+        end
+        if current_stat == "00000000" then
```
Comment (hossimo) [inline citp.lua]: Looks like this should evaluate to `supported_types`, not `current_stat`

**Comment by MrRoundRobin** [comment]
that's right, ugly copy & past error...

i thought about the "flag handling", and found out that this could look nicer if if you use `ProtoField`s.

i created a seperat branch and changed all fields to `ProtoField`. thats a [big change](https://github.com/hossimo/CITP-Dissector/compare/master...MrRoundRobin:fields?expand=1), but with that you can also filter on each field. if you like, i can create a pull request from that [branch](https://github.com/MrRoundRobin/CITP-Dissector/tree/fields)

**Pull request merged**